src/contracts/core/AllocationManager.sol

```diff
@@ -26,6 +26,13 @@ contract AllocationManager is
     using OperatorSetLib for OperatorSet;
     using SlashingLib for uint256;
 
+    modifier validateAVS(
+        address avs
+    ) {
+        require(isAVS(avs), InvalidAVS());
+        _;
+    }
+
```

Comment:
Would call this `onlyAVS` since it basically works like `onlyOwner` or `onlyOperator` or whatever.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/905/commits/4a8d82622e97b881672e3073f2e7796febf9200a